### DISCUSSION TITLE
custom osquery pack/queries ingested into SOC/Kibana

### DISCUSSION THREAD

**Discussion by Pancakesarusrex** [Security-Onion-Solutions/securityonion#5675] [Unsupported Versions] [answered]
when creating custom packs/queries through the fleet UI those results do not show up in kibana.  event.module: osquery. where do the packs/queries need to be specified in order for them to appear. in my setup fleet is running on the same instance as the manager.

**Comment by defensivedepth** [comment, marked as answer]
Can you confirm that you scheduled them to target online hosts? Can you also confirm that the custom queries will produce results ( ie run a Live Query with it)

**Reply by Pancakesarusrex** [reply]
i think there is just a delay from the customs queries/packs created through fleet UI. i was able to see the results in kibana after a day. to answer your questions yes the hosts were online and the queries produced results when running them through fleet. Now, im curious as to what causes the delay

**Reply by defensivedepth** [reply]
hmmm. What timeframe did you set the queries to run? every x minutes etc?

**Reply by Pancakesarusrex** [reply]
180 seconds

**Reply by defensivedepth** [reply]
hmm, that is odd. Please post to this discussion if you see it happening again.
